src/client/pages/Registration.tsx

```diff
@@ -140,7 +120,7 @@ export const Registration = ({
       >
         <form
           method="post"
-          action={`${Routes.REGISTRATION}?${registrationUrlQueryParamString}`}
+          action={`${Routes.REGISTRATION}?${queryString}`}
```

Comment:
It looks like queryString already includes the `?` separator 🤔

Reply:
Good spot!

```suggestion
          action={`${Routes.REGISTRATION}${queryString}`}
```